index.html

```diff
@@ -18,6 +18,11 @@
     </section>
     <section class="nav-menu">
         <nav class="navbar">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
```

Comment:
- Looking into sections `.nav-social` and `.nav-menu`, I'd like to suggest a few changes to improve your design here:
  - [ ] First, I will suggest that you have both sections in one `header` tag for semantic and accessibility reasons to indicate to browsers which part of the page the user is or where the content is located.
  - [ ] Secondly, the entire sections are not responsive as you will find out in this [video](https://watch.screencastify.com/v/o13Vaoq5PTINORZcHgIu). Kindly work on the responsiveness. You have done well by hiding the overflows but his look is not really good for user experience neither does it show your exceptional design skills to potential recruiters. :pleading_face: 
  - [ ] Third, the menu on a mobile screen is not good for user experience yet. Kindly remember how you implemented one for your portfolio not long ago. How there is a blurred background covering the entire viewport just so the navigations are seen well without conflicting with other parts of the page. Kindly work on this :pray:

Reply:
Good